assignment/assignment-impl/impl/src/java/org/sakaiproject/assignment/impl/AssignmentPeerAssessmentServiceImpl.java

```diff
@@ -262,10 +262,11 @@ private String findLowestAssignedAssessor(Map<String, Map<String, PeerAssessment
 		return lowestAssignedAssessor;
 	}
 
-	public List<PeerAssessmentItem> getPeerAssessmentItems(final Collection<String> submissionsIds){
+	public List<PeerAssessmentItem> getPeerAssessmentItems(final Collection<String> submissionsIds, Integer scaledFactor){
+		List<PeerAssessmentItem> listPeerAssessmentItem = new ArrayList<PeerAssessmentItem>();
```

Comment:
If this is not going to be merged into 10.x, you can use the diamond operator here.